builtin/providers/google/resource_compute_instance_template.go

```diff
@@ -134,7 +143,6 @@ func resourceComputeInstanceTemplate() *schema.Resource {
 			"automatic_restart": &schema.Schema{
 				Type:       schema.TypeBool,
 				Optional:   true,
-				Default:    true,
```

Comment:
@lwander you are correct - if we are going to change a default, then we may possibly need a state_migration